openapi/paths/path-item-with-examples.yaml

```diff
@@ -0,0 +1,41 @@
+post:
+  tags:
+    - Tag
+  summary: Operation summary with examples
+  description: |
+    Operation description **markdown**.
+  operationId: postPathItemWithExamples
+  security:
+    - api_key: []
+    - basic_auth: []
+  responses:
+    '200':
+      description: OK
+      headers:
+        X-Rate-Limit:
+          description: calls per hour allowed by the user
+          schema:
+            type: integer
+            format: int32
+        X-Expires-After:
+          $ref: ../components/headers/ExpiresAfter.yaml
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/Schema.yaml
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/Schema.yaml
+        examples:
```

Comment:
This is how to craft your own examples. While examples can use a `$ref` (reference object), keep in mind that object must have the structure like:

```yaml
summary: My example
description: | 
  Some Markdown **description**.
value:
   yourProp: abc
   another: xyz
```

**Many** people get confused and only include the value itself in the reference object:

```yaml
yourProp: abc
another: xyz
```
(This is incorrect.)